enterprise/frontend/src/embedding-sdk/docs/README.md

```diff
@@ -0,0 +1,197 @@
+
+```jsx
+import React from "react";
+import { MetabaseProvider, useQuestionSearch, InteractiveQuestion} from "@metabase/embedding-sdk-react";
+
+// Configuration
+const config = {
+  metabaseInstanceUrl: "https://metabase.example.com", // Required: Your Metabase instance URL
+  authType: "jwt", // Required
+  jwtProviderUri: "https://app.example.com/sso/metabase", // Required: Your endpoint that returns JWT token used to authenticate Metabase. We'll explain more below how to implement this endpoint.
+  font: "Lato", // Optional: you could provide any fonts support by Metabase
+}
+
+export default function App() {
+  const { data: questions, isLoading } = useQuestionSearch();
+
+  if (loading) {
+    return "Loading…"
+  }
+
+  // We just assume the logged in user have access to at least 1 question.
+  const firstQuestion = questions[0]
+
+  return (
+    <MetabaseProvider config={config}>
+      <InteractiveQuestion questionId={firstQuestion.id} />
+    </MetabaseProvider>
+  );
+}
+```
+
+`MetabaseProvider` also supports `pluginsConfig`. Currently we only allow configuring `mapQuestionClickActions`, but we'll support more plugins in next releases.
+
+```jsx
+// You can provide a custom action with your own `onClick` logic.
+const createCustomAction = clicked => ({
+  buttonType: "horizontal",
+  name: "client-custom-action",
+  section: "custom",
+  type: "custom",
+  icon: "chevronright",
+  title: "Hello from the click app!!!",
+  onClick: ({ closePopover }) => {
+    alert(`Clicked ${clicked.column?.name}: ${clicked.value}`);
+    closePopover();
+  },
+});
+
+// Or customize the appearance of the custom action to suit your need.
+const createCustomActionWithView = clicked => ({
+  name: "client-custom-action-2",
+  section: "custom",
+  type: "custom",
+  view: ({ closePopover }) => (
+    <button
+      className="tw-text-base tw-text-yellow-900 tw-bg-slate-400 tw-rounded-lg"
+      onClick={() => {
+        alert(`Clicked ${clicked.column?.name}: ${clicked.value}`);
+        closePopover();
+      }}
+    >
+      Custom element
+    </button>
+  ),
+});
+
+const plugins = {
+  /**
+   * You will have access to default `clickActions` that Metabase render by default.
+   * So you could decide if you want to add custom actions, remove certain actions, etc.
+   */
+  mapQuestionClickActions: (clickActions, clicked) => {
+    return [
+      ...clickActions,
+      createCustomAction(clicked),
+      createCustomActionWithView(clicked),
+    ]
+  }
+}
+
+return (
+  <MetabaseProvider config={config} pluginsConfig={plugins}>
+    <InteractiveQuestion questionId={firstQuestion.id}  />
+  </MetabaseProvider>
+);
+```
+
+After you have configured Metabase embedding SDK for React, you need to make sure your `jwtProviderUri` endpoint returns JWT token that your Metabase instance can use for authentication.
+
+**Metabase embedding SDK for React only support JWT authentication.**
```

Comment:
```suggestion
**Metabase Embedding SDK for React only supports JWT authentication**
```

On another note, we might want to put this on as a disclaimer at the top of the page, so users don't read everything only to realize they can't use it

Reply:
We have one up there
https://github.com/metabase/metabase/blob/ef02db2e0ee109875180692fc0acfddc4331f548/enterprise/frontend/src/embedding-sdk/docs/README.md?plain=1#L5

with all the typo lol. I'll fix the typo here and there as well.

Reply:
Ah sounds good